src/server/job_func.c

```diff
@@ -416,6 +416,57 @@ job_alloc(void)
 	return (pj);
 }
 
+#ifndef PBS_MOM
+
+/**
+ * @brief
+ * 	free work tasks and pending batch requests related to this job
+ *
+ * @param[in]	pj - pointer to job structure
+ *
+ * @return	void
+ */
+void
+free_job_work_tasks(job *pj)
+{
+	struct work_task *pwt;
+	struct batch_request *tbr = NULL;
+	/*
+	* Delete any work task entries associated with the job.
+	* mom deferred tasks via TPP are also hooked into the
+	* ji_svrtask now, so they also get automatically cleared
+	* in this following loop
+	*/
+	while ((pwt = (struct work_task *)GET_NEXT(pj->ji_svrtask)) != NULL) {
+		if (pwt->wt_type == WORK_Deferred_Reply) {
+			tbr = (struct batch_request *)pwt->wt_parm1;
+			if (tbr != NULL) {
+				/* Check if the reply is for scheduler
+					* If so, then reject the request.
+					*/
```

Comment:
Is the indentation off?  Github shows the second two lines of the comment indented further than the first.